markdoc.config.mjs

```diff
@@ -12,11 +12,43 @@ export default defineMarkdocConfig({
         title: { type: String, render: 'title' },
         mark: { type: Number, render: 'mark' }
       },
-      render: component('./src/components/CodeFence.astro'),
+      render: component('./src/components/Fence.astro'),
+    },
+    image: {
+      ...nodes.image,
+      render: component('./src/components/Figure.astro'),
+      attributes: {
+        src: { type: String },
+        alt: { type: String },
+        title: { type: String },
+        // https://discord.com/channels/830184174198718474/1182681722215743578/1204466454922002452
+        __optimizedSrc: { type: 'Object' },
+      },
     },
     link: {
       ...nodes.link,
       render: component('./src/components/Link.astro'),
     },
   },
+  tags: {
+    farmlandsSolutionCard: {
```

Comment:
Suggestion: This component could be something more generic. This is probably outside of the scope of your change so can leave it 🤷‍♀️